client/scss/overrides/_vendor.tagit.scss

```diff
@@ -82,10 +82,6 @@
     mask-image: url('#{$images-root}icons/cross.svg');
     mask-repeat: no-repeat;
   }
-
-  .ui-icon-close:hover:before {
-    color: theme('colors.critical.200');
-  }
```

Comment:
This icon still exists, but its color is defined with a `background-color`, so those styles do nothing.

Reply:
And I assume we do not want to use `colors.critical.200` for the icon?

Reply:
It’s a bit odd to me for sure.